File: API/facility-management-api/WebAPI/Models/Space.cs
using System.Text.Json.Serialization;

namespace WebAPI.Models
{
    public class Space
    {
        public int SpaceID { get; set; }
        public int FacilityID { get; set; }
        public string? SpaceName { get; set; }
        public int TypeID { get; set; }
        public int? Capacity { get; set; }
        public string? Status { get; set; }

        public Types? Type { get; set; }
        public ICollection<Asset> Assets { get; set; } = new List<Asset>();

        // Navigation Properties
        [JsonIgnore]
        public Facility? Facility { get; set; }

    }
}

File: Client/facility-management-app/src/models/Space.js
class Space{
  constructor(s){
    this.spaceId = s ? s.SpaceID : null;
    this.spaceName = s ? s.SpaceName : null;
    this.facilityID = s ? s.FacilityID : null;
    this.facilityName = s ? s.FacilityName : null;
    this.typeID = s && s.Type? s.Type.TypeID : null;
    this.typeName = s && s.Type? s.Type.TypeName : null;
    this.area = s ? s.Capacity : null;
  }
}

export default Space;